Lib/venv/__init__.py

```diff
@@ -409,8 +424,34 @@ def create(env_dir, system_site_packages=False, clear=False,
                          prompt=prompt, upgrade_deps=upgrade_deps)
     builder.create(env_dir)
 
+
+def get_default_args(venv_ini_default=VENV_INI_DEFAULT):
+    """ Check if a venv.ini exists in the platform location
+        + override defaults if it exists. Use `--help` to test ini config """
+    venvini_path = os.path.expanduser(venv_ini_default)
+    if not os.path.exists(venvini_path):
+        return CliDefaults()
+
+    cp = configparser.ConfigParser()
+    cp.read(venvini_path)
+    if not cp.has_section("venv"):
+        return CliDefaults()
+
+    arg_overloads = {}
+    for name, value in cp.items("venv"):
+        if value.lower() == "true":
+            arg_overloads[name] = True
+        elif value.lower() == "false":
+            arg_overloads[name] = False
+        else:
+            arg_overloads[name] = value
+
+    return CliDefaults(**arg_overloads)
```

Comment:
This logic will fail with a `TypeError` if the ini contains a variable which is not a valid tuple field name, or a typo in the value. As this is user-specified, untrusted input, checking needs to be bullet-proof.

Would redo as a loop over the known/expected field names, with checking for expected types, doing the 'true'/ True etc. conversions, checking for duplicates (raise `ValueError`) and so on.